### PR TITLE
Add checkconfig command

### PR DESCRIPTION
**Pull request by SLdragon**

### CODE REVIEW

File: iotedgehubdev/cli.py [hunk 1]
```diff
@@ -269,10 +269,23 @@ def stop(host):
     output.info('IoT Edge Simulator has been stopped successfully.')
 
 
+@click.command(context_settings=CONTEXT_SETTINGS,
+               help="Determine whether config file exists.")
+@_with_telemetry
+def checkconfig():
+    config_file = HostPlatform.get_config_file_path()
```
Comment (adashen) [inline iotedgehubdev/cli.py]: why not just call _parse_config_json?

Comment (SLdragon) [reply iotedgehubdev/cli.py]: updated

File: iotedgehubdev/cli.py [hunk 2]
```diff
@@ -269,10 +269,22 @@ def stop(host):
     output.info('IoT Edge Simulator has been stopped successfully.')
 
 
+@click.command(context_settings=CONTEXT_SETTINGS,
+               help="Determine whether config file is valid.")
+@_with_telemetry
+def validateconfig():
+    try:
```
Comment (adashen) [inline iotedgehubdev/cli.py]: no need to try/catch here. _parse_config_json already handle this right?

Comment (SLdragon) [reply iotedgehubdev/cli.py]: yes, but it contains other information, such as 'please run setup first'

Comment (SLdragon) [reply iotedgehubdev/cli.py]: Updated

**Pull request merged**